src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -70,7 +70,7 @@ export class QueryEnhancementsPlugin
       getQueryString: (currentQuery: Query) => `source = ${currentQuery.dataset?.title}`,
       fields: {
         sortable: false,
-        filterable: false,
+        filterable: true,
```

Comment:
we might have to revisit this as not every field can be filterable (non indexed fields)

Reply:
i think it takes the intersection, both language and the actual field need to be filterable in order to add filters
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/762eb4e5ca471f35f7194bd496820f26120b1e23/src/plugins/data/common/index_patterns/fields/index_pattern_field.ts#L135-L142

the field filterable behavior isn't modified

Reply:
actually you are right this is `getOsdFieldOverrides().filterable !== undefined` not `!getOsdFieldOverrides().filterable`, if setting true here overrides the check below. I removed `filterable: true,` in 0ee6f7e

also i pasted the wrong code, the logic to check whether fields are filterable for filter UI is in https://github.com/opensearch-project/OpenSearch-Dashboards/blob/0daad39467004cd06bd784ec154bcbb1b2e6ea63/src/plugins/data/common/index_patterns/fields/utils.ts#L48-L55